desktop/use-desktop/container.md

```diff
@@ -10,35 +10,28 @@ The **Containers** view lists all your running containers and applications. You
 
 Use the **Search** field to search for any specific container.
 
-From the **Containers** view you can perform the following actions on one or more containers at once:
+From the **Containers** view you can perform the following actions:
 - Pause/Resume
-- Stop/Start
+- Stop/Start/Restart
+- View image packages and CVEs
 - Delete
 - Open the application in VS code
 - Open the port exposed by the container in a browser
-- Copy docker run. This allows you to easily share container run details or modify certain parameters
+- Copy docker run. This lets you share container run details or modify certain parameters.
 
-### Integrated terminal
-
-You also have the option to open an integrated terminal, on a running container, directly within Docker Desktop. This allows you to quickly run commands within your container so you can understand its current state or debug when something goes wrong.
+## Resource usage
 
-Using the integrated terminal is the same as running `docker exec -it <container-id> /bin/sh`, or `docker exec -it <container-id> cmd.exe` if you are using Windows containers, in your external terminal. It also:
+From the **Containers** view you can monitor your containers' CPU and memory usage over time. This can help you understand if something is wrong with your containers or if you need to allocate additional resources. 
 
-- Automatically detects the default user for a running container from the image's Dockerfile. If no use is specified it defaults to `root`.
-- Persists your session if you navigate to another part of the Docker Dashboard and then return.
-- Supports copy, paste, search, and clearing your session.
-
-To open the integrated terminal, either:
-- Hover over your running container and select the **Show container actions** menu. From the drop-down menu, select **Open in terminal**.
-- Select the container and then select the **Terminal** tab.
-
- To use your external terminal, change your settings.
+When you [inspect a container](#inspect-a-container), the **Stats** tab displays further information about a container's resource utilization. You can see how much CPU, memory, network and disk space your container is using over time.
 
 ## Inspect a container
 
-You can obtain detailed information about the container when you select a container.
+You can obtain detailed information about the container when you select it.
+
+From here, you can use the quick action buttons to perform various actions such as pause, resume, start or stop, or explore the **Logs**, **Inspect**, **Bind mounts**, **Terminal**, **Files**, and **Stats** tabs. 
```

Comment:
In DD, I see **Exec** as the label for the terminal tab, not sure if that's worth calling out here but just thought I'd mention it in case someone new to development isn't familiar with this term.

Reply:
So I've triple checked this...the latest release candidate still shows 'terminal'. I suspect your on the latest internal release

Reply:
Ah that makes sense!